lnurl.py

```diff
@@ -146,11 +144,18 @@ async def api_lnurlw_callback(
 
     if paylink:
```

Comment:
nitpick:

```suggestion
    if not paylink:
        return {"status": "ERROR", "reason": "no paylink found"}
```

Reply:
you mean revert the logic and return error first. i can change it...

Reply:
yes, easier to read
and one less indentation level